### PR TITLE
Security Domain property added to several steps

### PR DESCRIPTION
**Pull request by daviddelapena**
Fix for Issue #28.

Property "Security Domain" added to following steps:

-Run PMREP Command
-Create Static Deployment group
-Create Dynamic Deployment group
-Deploy Deployment group
-Roll Back Deployment group

### CODE REVIEW

File: src/main/scripts/informatica_deploy_deployment.groovy [hunk 1]
```diff
@@ -92,13 +94,23 @@ println('')
 
 def script = new File(inputFile)
 script << "connect -r $srcrepo -n $srcusername -x $srcpassword "
+
+if (srcsecurityDomain){
```
Comment (nhmathis) [inline src/main/scripts/informatica_deploy_deployment.groovy:98]: Do both of these properties use the -s flag? Are they supposed to? 

If they do, what happens if they are both set? Does the second override the other?

Comment (daviddelapena) [reply src/main/scripts/informatica_deploy_deployment.groovy:98]: The command looks something like :
connect -r ... -n ... -x ... -s ... deploydeploymentgroup -p ... -c ... -r ... -n ... -x ... -s ... 
you can see that most flags are duplicated.

Looking at some doc of deploydeploymentgroup command:

pmrep DeployDeploymentGroup
The DeployDeploymentGroup command uses the following syntax:
deploydeploymentgroup
-p <deployment_group_name>
-c <control_file_name>
-r <target_repository_name>
[-n <target_repository_user_name>
[-s <target_repository_user_security_domain>]
 [-x <target_repository_password> |
 -X <target_repository_password_environment_variable>]
 [-d <target_domain_name> |
 {-h <target_portal_host_name>
 -o <target_portal_port_number>}]] (only if target is in a differentdomain)
[-l <log_file_name>]

link:
https://kb.informatica.com/h2l/HowTo%20Library/1/0083-DeploymentUsingPmrepCommands.pdf

Comment (nhmathis) [reply src/main/scripts/informatica_deploy_deployment.groovy:98]: Gotcha thanks!

File: plugin/upgrade.xml [hunk 2]
```diff
@@ -218,4 +218,32 @@
...
+      </migrate-command>
+      <migrate-command name="Create Static Deployment group">
+          <migrate-properties>
+              <migrate-property name="securityDomain" default=""/>
+          </migrate-properties>
+      </migrate-command>
+      <migrate-command name="Deploy Deployment group">
+          <migrate-properties>
+              <migrate-property name="srcsecurityDomain" default=""/>
+              <migrate-property name="securityDomain" default=""/>
```
Comment (nhmathis) [inline plugin/upgrade.xml:235]: For the future, you can migrate property names, for example
`<migrate-property name="tarsecurityDomain" old="securityDomain" default=""/>`

Comment (daviddelapena) [reply plugin/upgrade.xml:235]: Thanks! But in this case there is only new properties as all Security Domain props where missing.

**Pull request merged**